templates/ssl_ratios_raw_v1.sql

```diff
@@ -0,0 +1,24 @@
+SELECT
+  submission_date,
+  os,
+  country,
+  SUM(udf_get_key(histogram_parent_http_pageload_is_ssl, 0)) AS non_ssl_loads,
+  SUM(udf_get_key(histogram_parent_http_pageload_is_ssl, 1)) AS ssl_loads,
```

Comment:
`histogram_parent_http_pageload_is_ssl` is a map and may contain a variable number of rows in any order, so `histogram_parent_http_pageload_is_ssl[SAFE_OFFSET(0)].value` would not always return the correct value.

the correct thing to do here is use `udf_get_key` instead.